source/_integrations/binary_sensor.rest.markdown

```diff
@@ -49,13 +49,25 @@ binary_sensor:
     resource: http://IP_ADDRESS/ENDPOINT
     method: POST
 ```
+or a template based request:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: rest
+    resource_template: {% raw %} "http://IP_ADDRESS/{{ now().strftime('%Y-%m-%d') }}" {% endraw %}
+```
 
 {% configuration %}
 resource:
   description: The resource or endpoint that contains the value.
   required: true
   type: string
   default: string
+resource_template:
+  description: The resource or endpoint that contains the value with template support.
+  required: true
```

Comment:
It is not required according to the codebase.

Reply:
changed to required: false